tests/common/TestRuntime.cs

```diff
@@ -1449,6 +1453,18 @@ public static void IgnoreInCIfHttpStatusCodes (Exception ex, params HttpStatusCo
 		IgnoreInCI ($"Ignored due to http status code '{status}': {ex.Message}");
 	}
 
+	public static void IgnoreInCIIfNetworkConnectionLost (Exception ex)
+	{
+		// <Foundation.NSErrorException: Error Domain=NSURLErrorDomain Code=-1005 "The network connection was lost." UserInfo ...
+		if (!(ex is NSErrorException nex))
```

Comment:
Why not using 'is not'. Works the same and will set nex for the follow up code. 
```suggestion
		if (ex is not NSErrorException nex)
```